spec/models/principal_spec.rb

```diff
@@ -19,24 +19,22 @@
 
 require File.dirname(__FILE__) + '/../spec_helper'
 
-describe Principal, :type => :model do
-  describe "ATTRIBUTES" do
+describe Principal, type: :model do
+  describe 'ATTRIBUTES' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.